staging/src/k8s.io/apiextensions-apiserver/test/integration/validation_test.go

```diff
@@ -203,7 +203,7 @@ func TestCustomResourceUpdateValidation(t *testing.T) {
 	}
 	defer close(stopCh)
 
-	// enable alpha feature CustomResourceValidation
+	// enable feature CustomResourceValidation
```

Comment:
There are a few other places in this file with this line. Can you update them too? :)

Reply:
done